perf_test/sparse/KokkosSparse_spmv_test.hpp

```diff
@@ -33,6 +33,9 @@
 #include <spmv/MKL_SPMV.hpp>
 #endif
 
+template <typename AType, typename XType, typename YType>
+void armpl_matvec(AType /*A*/, XType x, YType y, spmv_additional_data* data);
```

Comment:
Just curious: why is A commented out but others are not?

Reply:
@ajpowelsnl : can you comment?

Reply:
@ajpowelsnl I am awaiting your comment.